DiverseSelector/methods/dissimilarity.py

```diff
@@ -55,62 +61,71 @@ def __init__(self, func_distance=None):
         Parameters
         ----------
         func_distance: callable
-            Function for calculating the pairwise distance between instances of the array.
+            Function for calculating the pairwise distance between instances of the coordinates array.
         """
         self.func_distance = func_distance
 
-    def select_from_cluster(self, arr, num_selected, cluster_ids=None):
-        """
-        Algorithm MaxMin for selecting points from cluster.
+    def select_from_cluster(self, X, size, cluster_ids=None):
+        """Return selected samples from a cluster based on MaxMin algorithm.
 
         Parameters
         ----------
-        arr: np.ndarray
+        X: np.ndarray
```

Comment:
The problem above, but I will not mention it again in the following comments to keep everything tiny and readable.